schema/codelists/closed/process_type.csv

```diff
@@ -7,7 +7,7 @@ hydrological_drought,Hydrological Drought,Occurs when low water supply becomes e
 meteorological_drought,Meteorological Drought,Occurs when dry weather patterns dominate an area. It is defined usually on the basis of the degree of dryness and the duration of the dry period.
 socioeconomic_drought,Socio-economic Drought,"Relates the supply and demand of some economic goods with elements of meteorological, hydrological, and agricultural drought. It also occurs when the demand for an economic good exceeds supply as a result of a weather-related shortfall in water supply."
 primary_rupture,Primary Rupture,"Earthquake surface ruptures and fissures are localised ground displacements that develop during and immediately after an earthquake. Surface ruptures represent the upward continuation of fault slip at depth, while fissures are smaller displacements, or more distributed deformation in and around the rupture area."
-secondary_rupture,Secondary Rupture,TBA
+secondary_rupture,Secondary Rupture,"Secondary earthquake ruptures occur due to stress tranfser from a primary rupture, causing concurrent or subsequent rupture of intersecting faults in the same sin a fault set."
```

Comment:
```suggestion
secondary_rupture,Secondary Rupture,"Earthquake ruptures that occur due to stress transfer from a primary rupture, causing concurrent or subsequent rupture of intersecting faults in the same sin a fault set."
```
> same sin a fault set

Is this final bit of the sentence a typo?

Reply:
should read: 
secondary_rupture,Secondary Rupture,"Earthquake ruptures that occur due to stress transfer from a primary rupture, causing concurrent or subsequent rupture of intersecting faults in the same fault set."